config.go

```diff
@@ -87,16 +118,19 @@ func (config TGFConfig) String() (result string) {
 	ifNotZero(dockerImage, config.Image)
 	ifNotZero(dockerImageVersion, config.ImageVersion)
 	ifNotZero(dockerImageTag, config.ImageTag)
-	ifNotZero(dockerImageBuildFolder, config.ImageBuildFolder)
-	if config.ImageBuild != "" {
-		lines := strings.Split(strings.TrimSpace(config.ImageBuild), "\n")
-		buildScript := lines[0]
-		if len(lines) > 1 {
-			sep := "\n    "
-			buildScript = sep + strings.Join(lines, sep)
-		}
+	for _, ib := range config.ImageBuild {
+		ifNotZero(ib.Folder, ib.Folder)
```

Comment:
Maybe something like "docker-image-build.index" should be given instead of `ib.Folder` twice

Reply:
Wrong code, thank you!